### ISSUE TITLE
Wrong Portuguese from Portugal translation

### ISSUE THREAD

**Issue by jfreves** [rejuvenate/lovelace-horizon-card#135]
When setting up the language with `pt` (Portuguese from Portugal) the translation for moonrise and moonset ir not correct, it is using dawn and dusk values which are not correct.
<img width="386" alt="image" src="https://github.com/rejuvenate/lovelace-horizon-card/assets/75815666/287a64ad-96c8-42e6-8b50-c8f0e731ef44">

```
"dawn": "Amanhecer",
"dusk": "Anoitecer",
"elevation": "Elevação",
"moonrise": "Anoitecer",
"moonset": "Amanhecer",
```

It seems that was already [a PR with the correct translation](https://github.com/rejuvenate/lovelace-horizon-card/pull/77/files) but was dismissed for some reason.

**Comment by ThomDietrich** [comment]
Hey @jfreves, not quite the whole story. Please see https://github.com/rejuvenate/lovelace-horizon-card/pull/100
CC @ViPeR5000

**Comment by jfreves** [comment]
> Hey @jfreves, not quite the whole story. Please see #100 CC @ViPeR5000 Pleae open a PR with your fix to all the above.

I noticed that PR but don't understand the reason for it since the one from FragMenthor was correct.

**Comment by ThomDietrich** [comment]
You're right, moonrise/moonset were using the dusk/dawn words. Fixed in pt ("Nascer da lua" / "Pôr da lua") in #206. Thanks!
